README.md

```diff
@@ -1,26 +1,102 @@
-# gatsby-starter-default
-The default Gatsby starter.
+<p align="center">
+  <a href="https://next.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Gatsby's default starter
+</h1>
 
-For an overview of the project structure please refer to the [Gatsby documentation - Building with Components](https://www.gatsbyjs.org/docs/building-with-components/).
+Kick off your your project with this default boilerplate. This barebones starter ships with the main Gatsby configuration files you might need. 
 
-## Install
+_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://next.gatsbyjs.org/docs/gatsby-starters/)._
 
-Make sure that you have the Gatsby CLI program installed:
-```sh
-npm install --global gatsby-cli
-```
+## 🚀 Quick start
 
-And run from your CLI:
-```sh
-gatsby new gatsby-example-site
-```
+1.  **Install the Gatsby CLI.**
 
-Then you can run it by:
-```sh
-cd gatsby-example-site
-gatsby develop
-```
+    The Gatsby CLI helps you create new sites using Gatsby starters (like this one!)
 
-## Deploy
+    ```sh
+    # install the Gatsby CLI globally
+    npm install -g gatsby-cli
+    ```
+
+2.  **Create a Gatsby site.**
+
+    Use the Gatsby CLI to create a new site, specifying the default starter.
+
+    ```sh
+    # create a new Gatsby site using the default starter
+    gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default#v2
+    ```
+
+3.  **Start developing.**
+
+    Navigate into your new site’s directory and start it up.
+
+    ```sh
+    cd my-default-starter/
+    gatsby develop
+    ```
+
+4.  **Open the source code and start editing!**
+
+    Your site is now running at `http://localhost:8000`. Open the the `my-default-starter` directory in your code editor of choice and edit `src/pages/index.js`. Save your changes and the browser will update in real time!
```

Comment:
Wondering if it's worth mentioning the Graphiql URL here too http://localhost:8000___graphql, b/c even though the command line suggests it, some people might not know what it is / ignore it?

Reply:
I'll add an note mentioning that it's something we'll dive into later, but not to worry about it for now